src/pathfinding_service/api.py

```diff
@@ -468,8 +468,15 @@ def get(self, checksummed_address: str) -> Tuple[Dict[str, Union[str, PeerCapabi
                 UserPresence.ONLINE,
                 UserPresence.UNAVAILABLE,
             ]:
+                displayname = user_manager._displayname_cache.userid_to_displayname.get(
+                    user_id, None
+                )
```

Comment:
Could be shortened to
```suggestion
                displayname = user_manager._displayname_cache.userid_to_displayname.get(
                    user_id
                )
```

Reply:
I added the default on purpose to make it explicit that this will be the returned value in case this user ID is not present. Do you think this is not useful?

Reply:
It's not helpful for me personally, but I don't mind either way. It clearly is nitpicking.